app/helpers/decidim/department_admin/application_helper.rb

```diff
@@ -37,6 +41,14 @@ def user_assemblies_filtered(user, _locale, search_text)
         query = query.where("lower(title->>?) like lower(?)", current_locale, "%#{search_text}%") if search_text.present?
         query
       end
+
+      def user_conferences_filtered(user, _locale, search_text)
+        return unless defined?(Decidim::Conferences)
```

Comment:
sorry I think returning `nil` will produce a null exception in line 24. Better return an empty array:
```suggestion
        return [] unless defined?(Decidim::Conferences)
```